packages/stark-ui/src/modules/table/components/table.component.ts

```diff
@@ -1121,6 +1151,14 @@ export class StarkTableComponent extends AbstractStarkUiComponent implements OnI
 		}
 	}
 
+	/**
+	 * Collapse a row
+	 * @param row - Row to collapse
+	 */
+	public expandRow(row: object): void {
```

Comment:
Could you please propose an alternative to this ? 
I'm not sure that forcing developers to use
```typescript
@ViewChild(StarkTableComponent)
public tableComponent: StarkTableComponent;

public handleRowClicked(row: any): void {
	this.tableComponent.expandRow(row);
}
```

is the best idea since it forces them to use internal methods of the Table component (something we try to avoid as much as possible)